lib/proc/proc_select.cpp

```diff
@@ -4914,6 +4914,10 @@ namespace {
       if (!grn_select_post_filter(ctx, data)) {
         goto exit;
       }
+      if (data->filter.post_filter.length != 0) {
```

Comment:
I think we had better execute this line when using `post_filter` option.
I'm not sure we should skip it here or inside the `grn_select_post_filter` function.